connect/cli/plugins/project/extension/helpers.py

```diff
@@ -34,8 +37,20 @@ def bootstrap_extension_project(config, output_dir, overwrite):  # noqa: CCR001
         statuses_by_event[elem['extension_type']][elem['type']] = elem['object_statuses']
         grouped_definitions[elem['extension_type']][elem['category']].append(elem)
 
+    questions = get_questions(config, grouped_definitions)
+
+    if load_answers_dir and os.path.splitext(load_answers_dir)[-1] == '.json':
+        with open(load_answers_dir) as fp:
+            loaded_answers = json.load(fp)
+
+        for question in questions:
+            if question['name'] in loaded_answers:
```

Comment:
it's a great idea to set the defaults but probably we should open the wizard directly showing the summary page.

Reply:
No really understand, without ability to change anything in wizard?

Reply:
No ,you can always from the summary go back